packages/ui-components/src/components/tag/tag.tsx

```diff
@@ -0,0 +1,22 @@
+import { Component, Host, Prop, h } from '@stencil/core';
+
+@Component({
+	tag: 'kv-tag',
+	styleUrl: 'tag.scss',
+	shadow: true
+})
+export class KvTag {
+	/** (optional) Tag text */
```

Comment:
Should be optional or required??

Reply:
why is it optional?

Reply:
Optional because imagine a use case where the user wants to set a different Typography, He can do it with the slot and discard the usage of the text.